e2e/test_reconnection.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+
+case "$1" in
+  rest)
+    rest_arg="--rest"
+    server_app_address="http://localhost:9091"
+    server_address="http://localhost:9093"
+    db_arg="--database :flwr-in-memory-state:"
+    ;;
+  sqlite)
+    rest_arg=""
+    server_address="127.0.0.1:9092"
+    server_app_address="127.0.0.1:9091"
+    db_arg="--database $(date +%s).db"
+    ;;
+  *)
+    rest_arg=""
+    server_address="127.0.0.1:9092"
+    server_app_address="127.0.0.1:9091"
+    db_arg="--database :flwr-in-memory-state:"
+    ;;
+esac
+
+dir_arg="--dir ./.."
+
+timeout 2m flower-superlink --insecure $db_arg $rest_arg &
+sl_pid=$!
+sleep 3
+
+timeout 2m flower-client-app client:app --insecure $rest_arg --server $server_address &
+cl1_pid=$!
+sleep 3
+
+timeout 2m flower-client-app client:app --insecure $rest_arg --server $server_address &
+cl2_pid=$!
+sleep 3
+
```

Comment:
I think it could be useful to echo a message here, so that we know what stage of the test we are at?